### PR TITLE
docs(spec): consolidate disambiguation spec

### PR DESCRIPTION
**Pull request by bdarcus**
## Summary

- Create `docs/specs/DISAMBIGUATION.md` — normative design spec consolidating two stale architecture docs
- Delete `DISAMBIGUATION_IMPLEMENTATION_PLAN.md` (4/11-test tracker, now 11/11 and obsolete) and `DISAMBIGUATION_MULTILINGUAL_GROUPING.md` (migrated in full)
- Addresses two corner cases flagged by CSL schema maintainers ([styles#7667](https://github.com/citation-style-language/styles/issues/7667), [schema#452](https://github.com/citation-style-language/schema/issues/452))

## Scope

**New:** `docs/specs/DISAMBIGUATION.md`  
**Deleted:** `docs/architecture/DISAMBIGUATION_IMPLEMENTATION_PLAN.md`, `docs/architecture/DISAMBIGUATION_MULTILINGUAL_GROUPING.md`  
**Updated cross-refs:** `DESIGN_PRINCIPLES.md`, `ARCHITECTURAL_SOUNDNESS_2026-02-25.md`, `MULTILINGUAL_GROUPING_STYLE_TARGETS.md`, `docs/reference/DISAMBIGUATION.md`, `docs/specs/README.md`  
**New beans:** `csl26-6ncw`, `csl26-zrz5`, `csl26-xrc5`

## Key design decisions

**Case A (APA §8.15 — year-suffix with differing original dates):** Citum already produces the correct output by design — the collision key uses only `issued` year (`build_group_key`), never the original-publication date. citeproc-js is the diverging implementation. No legacy-parity toggle is added.

**Case B (schema#452 — generalized ignore-for-disambiguation):** Designed as a `disambiguate.ignore: [original-date]` option on `Disambiguation` struct (no template-language attribute). Spec section locked; implementation deferred to `csl26-zrz5`.

## Validation

Docs-only change — Rust gate not required per CLAUDE.md doc-only rule. Pre-push hook confirms no Rust files changed.

Link check: `grep -rn "DISAMBIGUATION_IMPLEMENTATION_PLAN\|DISAMBIGUATION_MULTILINGUAL_GROUPING" docs/` returns only two changelog lines in the new spec (intentional historical references, not broken links).

## Risk

None — no code changes. The two deleted docs were superseded by live implementation and the new spec; git history preserves them.

## Follow-ups

- `csl26-6ncw` — native test fixture asserting `(1926/1967a) (1926/1967b) (1927/1967c)`
- `csl26-zrz5` — implement `disambiguate.ignore` option (schema + engine + schema regen)
- `csl26-xrc5` — short-title-for-disambiguation design (draft, larger scope)

### CODE REVIEW

File: .beans/csl26-xrc5--short-title-for-disambiguation-first-reference-not.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# csl26-xrc5
+title: Short-title-for-disambiguation + first-reference-note-number (CSL schema#452 part B)
+status: draft
+type: task
+created_at: 2026-05-29T11:15:03Z
+updated_at: 2026-05-29T11:15:03Z
+---
+
+Design and implement a disambiguation strategy that can render a short title *only* when needed to disambiguate, compatible with `first-reference-note-number` cross-references (CSL schema#452 part B).\n\nCitum has note-number support (`processor/document/note_support.rs`) but no short-title-for-disambiguation strategy. This is larger than the `disambiguate.ignore` work (csl26-zrz5); treat as a separate feature.\n\nSpec context: docs/specs/DISAMBIGUATION.md §6 (open question).\n\nRef: https://github.com/citation-style-language/schema/issues/452
```
Comment (bdarcus) [inline .beans/csl26-xrc5--short-title-for-disambiguation-first-reference-not.md]: No `\n` in files, particularly markdown

Also, link to the actual github issue and/or zotero forum thread, which is collected in that issue, for clarity.

File: .beans/csl26-zrz5--case-b-implement-disambiguateignore-signature-excl.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+# csl26-zrz5
+title: 'Case B: implement `disambiguate.ignore` signature exclusion option'
+status: todo
+type: task
+created_at: 2026-05-29T11:14:56Z
+updated_at: 2026-05-29T11:14:56Z
+---
+
+Implement the `disambiguate.ignore` list option specified in docs/specs/DISAMBIGUATION.md §6.\n\n## Steps\n\n- [ ] Add `ignore: Option<Vec<ReferenceVariable>>` to the `Disambiguation` struct in `crates/citum-schema-style/src/options/processing.rs:393`\n- [ ] Thread into `DisambiguationFlags` and plumb into `build_group_key` in `crates/citum-engine/src/processor/disambiguation.rs`\n- [ ] Regenerate JSON schemas: `cargo run --bin citum --features schema -- schema --out-dir docs/schemas`\n- [ ] Add tests for `ignore: [original-date]` variant\n- [ ] Set spec status to Active
```
Comment (bdarcus) [inline .beans/csl26-zrz5--case-b-implement-disambiguateignore-signature-excl.md]: more `\n`

File: .beans/csl26-6ncw--case-a-fixture-asserting-issued-year-only-year-suf.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+# csl26-6ncw
+title: 'Case A: fixture asserting issued-year-only year-suffix (APA §8.15 reprint scenario)'
+status: todo
+type: task
+created_at: 2026-05-29T11:14:49Z
+updated_at: 2026-05-29T11:14:49Z
+---
+
+Add a native integration test in `crates/citum-engine/tests/citations.rs` that asserts the three-reprint scenario produces `(1926/1967a) (1926/1967b) (1927/1967c)`. Three references, same author, issued 1967, original-dates 1926/1926/1927. Citum already produces the correct output by design; this fixture locks it against regression.\n\nSpec: docs/specs/DISAMBIGUATION.md §1 (Case A).
```
Comment (bdarcus) [inline .beans/csl26-6ncw--case-a-fixture-asserting-issued-year-only-year-suf.md]: more `\n`

File: docs/specs/DISAMBIGUATION.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+  and a "Books" group can each begin `(2020a) (2020b)` independently.
+- **Group sort** (if set) drives suffix ordering instead of the global title
+  sort.
+- No suffix escapes a group's boundary; within-group collision detection is
+  scoped to that group's reference set.
+
+Without `disambiguate: locally`, disambiguation runs globally across the full
+bibliography.
+
+### 6. Case B — `disambiguate.ignore` option (design, not yet implemented)
```
Comment (bdarcus) [inline docs/specs/DISAMBIGUATION.md]: "Case B" means nothing in this context

File: docs/specs/DISAMBIGUATION.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+**Implementation path (deferred):**
+1. Add `ignore: Option<Vec<ReferenceVariable>>` to `Disambiguation` in
+   `crates/citum-schema-style/src/options/processing.rs:393`.
+2. Thread into `DisambiguationFlags` and plumb to `build_group_key`.
+3. Regenerate JSON schemas (`cargo run --bin citum --features schema -- schema
+   --out-dir docs/schemas`).
+
+#### Sub-case: short-title + `first-reference-note-number` (open question)
+
+schema#452 also describes a scenario where a style uses a short title *only* for
```
Comment (bdarcus) [inline docs/specs/DISAMBIGUATION.md]: use actual links, and per above, make sure it's specific issue/thread
